x-pack/test/functional/apps/maps/group1/layer_visibility.js

```diff
@@ -31,6 +32,7 @@ export default function ({ getPageObjects, getService }) {
 
     it('should fetch layer data when layer is made visible', async () => {
       await PageObjects.maps.toggleLayerVisibility('logstash');
+      await testSubjects.click('mapLayerTOC'); // Tooltip blocks clicks otherwise
```

Comment:
Thanks for updating to EUI v55.1.3. This function test change, requiring clicking legend to close the layer tooltip highlights a poor user experience introduced with this EUI update. I have captured the experience in screen shots below.
1) Layer is not visible
    <img width="300" alt="Screen Shot 2022-05-19 at 8 31 55 AM" src="https://user-images.githubusercontent.com/373691/169321178-935f944e-167e-489c-a209-4148bf38305d.png">
2) User opens actions menu and clicks "Show layer"
    <img width="300" alt="Screen Shot 2022-05-19 at 8 32 01 AM" src="https://user-images.githubusercontent.com/373691/169321537-d831e3cb-59be-40fe-97aa-93323d48fc0f.png">
3) Layer is now visible - but why is this tooltip opened? This is unintended behavior. How can we resolve this?
    <img width="300" alt="Screen Shot 2022-05-19 at 8 32 06 AM" src="https://user-images.githubusercontent.com/373691/169321847-cafaa99c-760e-43fb-986d-7ae9577865c5.png">

Reply:
@nreese It's actually not unexpected behavior. It's all about accesibility:

 - When closing context menus/popovers, the focus is returned to the triggering button. This keeps keyboard users in the same context from where they came instead of back at the beginning of the dom.
 - Tooltips show on focus, not just hover (again, required for keyboard users). So when the user's focus is put back on the triggering button, the tooltip will show since the focus is on that element.

Based on those concepts do you have a proposed solution to handle this scenario?

Reply:
Thanks for listing the accessibility behavior. That helps frame the problem.

> Based on those concepts do you have a proposed solution to handle this scenario?
I am not sure on a the best path forward. Having the tooltip open when clicking an action is a poor (annoying) user experience and should not happen. I am not sure how to get around the issue with the accessibility behavior listed above. Do you have any ideas?

Reply:
Nothing that will be resolved during this upgrade unfortunately.

Reply:
> Having the tooltip open when clicking an action is a poor (annoying) user experience and should not happen.

This is a mouse/desktop user centric POV that doesn't account for touchscreen users, keyboard users, or screen reader users. Only showing a tooltip on hover doesn't allow for any of the above alternate forms of interacting with the UI. We need to ensure that tooltips are accessible and appear on click/focus.

My 2c: I would either suggest leaving this behavior as is (since this patch is a fix and how it should have behaved all along), _or_ not wrapping the tooltip around the entire popover toggle button. You could add another blue icon to the right of the popover toggle that has an "info" icon that triggers the tooltip.

Reply:
> My 2c: I would either suggest leaving this behavior as is (since this patch is a fix and how it should have behaved all along), or not wrapping the tooltip around the entire popover toggle button. You could add another blue icon to the right of the popover toggle that has an "info" icon that triggers the tooltip.

Thanks, I will open an issue and Maps will move the tooltip in a follow up PR.

Reply:
created https://github.com/elastic/kibana/issues/132554

Reply:
Thanks @nreese!